blob/fileblob/fileblob.go

```diff
@@ -86,10 +86,10 @@ func openBucket(dir string, _ *Options) (driver.Bucket, error) {
 	dir = filepath.Clean(dir)
 	info, err := os.Stat(dir)
 	if err != nil {
-		return nil, fmt.Errorf("open file bucket: %v", err)
+		return nil, err
 	}
 	if !info.IsDir() {
-		return nil, fmt.Errorf("open file bucket: %s is not a directory", dir)
+		return nil, fmt.Errorf("%s is not a directory", dir)
```

Comment:
I don't see how removing "open file bucket" helps your purpose.

Reply:
It doesn't really, but I was trying to make the error messages more consistent. They used to all have this kind of prefix, or at least mostly, but now that we're usually returning raw errors, they don't.